docusaurus/docs/extensions/extensions-configuration.md

```diff
@@ -0,0 +1,956 @@
+    path:      `/${ YOUR_PRODUCT_NAME }/c/:cluster/${ CUSTOM_PAGE_NAME }`,
+    component: MyCustomPage,
+    meta:      {
+      product: YOUR_PRODUCT_NAME,
+      cluster: BLANK_CLUSTER
+    },
+  },
+  {
+    name:      `${ YOUR_PRODUCT_NAME }-c-cluster-resource`,
+    path:      `/${ YOUR_PRODUCT_NAME }/c/:cluster/:resource`,
+    component: ListResource,
+    meta:      {
+      product: YOUR_PRODUCT_NAME,
+      cluster: BLANK_CLUSTER
+    },
+  },
+  {
+    name:      `${ YOUR_PRODUCT_NAME }-c-cluster-resource-create`,
+    path:      `/${ YOUR_PRODUCT_NAME }/c/:cluster/:resource/create`,
+    component: CreateResource,
+    meta:      {
+      product: YOUR_PRODUCT_NAME,
+      cluster: BLANK_CLUSTER
+    },
+  },
+  {
+    name:      `${ YOUR_PRODUCT_NAME }-c-cluster-resource-id`,
+    path:      `/${ YOUR_PRODUCT_NAME }/c/:cluster/:resource/:id`,
+    component: ViewResource,
+    meta:      {
+      product: YOUR_PRODUCT_NAME,
+      cluster: BLANK_CLUSTER
+    },
+  },
+  {
+    name:      `${ YOUR_PRODUCT_NAME }-c-cluster-resource-namespace-id`,
+    path:      `/${ YOUR_PRODUCT_NAME }/:cluster/:resource/:namespace/:id`,
+    component: ViewNamespacedResource,
+    meta:      {
+      product: YOUR_PRODUCT_NAME,
+      cluster: BLANK_CLUSTER
+    },
+  }
+];
+
+export default routes;
+```
+
+> Note: Noticed that we didn't need to define the parameter `resource` under `meta`? Since it a wildcarded parameter on the path and it's not mandatory like `cluster`, we don't need to define it on the routes definition.
+
+On the above routes definition for `YOUR_K8S_RESOURCE_NAME` the user will get the default list view automatically wired in to display the list of `YOUR_K8S_RESOURCE_NAME` instances (`${ YOUR_PRODUCT_NAME }-c-cluster-resource`).
+
+The remaining routes will ensure that all the necessary connections are done for create/edit views, but they will not provide any interfaces for those view types! Those will have to be created by the developer and placed on folders with the correct naming in order to make them work.
+
+Let's then look at an example of this:
+
+```ts
+const YOUR_K8S_RESOURCE_NAME = 'your-custom-crd-name';
+```
+
+if a user wants a custom `list` view for the resource `your-custom-crd-name`, one will need to create a folder called `list` inside your extension folder, and the create file there for a vue component called `your-custom-crd-name.vue`. By following this pattern, Rancher Dashboard will take care of the wiring for you.
+
+For a `detail` view, just create a folder called `detail` inside your extension folder, and the create file there for a vue component called `your-custom-crd-name.vue`.
+
+For an `edit` view, just create a folder called `edit` inside your extension folder, and the create file there for a vue component called `your-custom-crd-name.vue`. The edit will dub as a `create` view also, so need to add a `create` folder. It can even dub as a `detail` view if you don't wish to duplicate it. 
```

Comment:
The second to last sentence should read "The edit will dub as a `create` view also, so ***no*** need to add...".

Reply:
fixed